### PR TITLE
WEBUI-222: bump to TP 2021.9

### PR DESCRIPTION
**Pull request by guirenard**

### CODE REVIEW

**Review by semisse — DISMISSED** [review]

**Review by richardsd — DISMISSED** [review]

**Review by semisse — DISMISSED** [review]

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.nuxeo</groupId>
     <artifactId>nuxeo-parent</artifactId>
-    <version>2021.6</version>
+    <version>2021.7</version>
```
Comment (richardsd) [inline pom.xml]: Couldn't we set the version to **2021.8** (at least until the 2021.9 is promoted)?

File: package.json [hunk 2]
```diff
@@ -180,6 +180,7 @@
     "dotenv": "^8.0.0",
     "es6-promise": "^4.0.0",
     "extend": "^3.0.2",
+    "inquirer": "8.1.2",
```
Comment (richardsd) [inline package.json]: Why do we need to add this dependency? Do you know if it was coming from another package already?

**Review by richardsd — DISMISSED** [review]

File: package.json (hunk 2 above)
Comment (Gabez0r) [reply package.json]: What do we need this package for?

Comment (guirenard) [reply package.json]: Please see https://jira.nuxeo.com/browse/WEBUI-474 referenced in the associated commit -> https://github.com/SBoudrias/Inquirer.js/issues/1046
It is now fixed and they released a newer version but at the time I couldn't get the build passing. I am gonna revert this commit now

**Review by semisse — APPROVED** [review]

**Review by doulba — APPROVED** [review]

**Pull request merged**